agr_indexer/src/main/java/org/alliancegenome/indexer/indexers/curation/DiseaseAnnotationCurationIndexer.java

```diff
@@ -399,10 +376,23 @@ private void populateBaseDiseaseAnnotationDocument(BiologicalEntity biologicalEn
 	}
 
 	private static String getGeneticModifierConsolidatedKey(DiseaseAnnotation da) {
-		if (da.getDiseaseGeneticModifierRelation() == null && CollectionUtils.isEmpty(da.getDiseaseGeneticModifiers())) {
+		if (da.getDiseaseGeneticModifierRelation() == null &&
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.OperatorWrapCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'&&' should be on a new line.
<!-- __reviewdog__:ChBjZjk3OWRhMzQyNGFjNzY1EgpjaGVja3N0eWxl -->